bbdo/bam/dimension_kpi_event.cc

```diff
@@ -16,7 +16,7 @@
 ** For more information : contact@centreon.com
 */
 
-#include "com/centreon/broker/bam/dimension_kpi_event.hh"
+#include "bbdo/bam/dimension_kpi_event.hh"
 
 #include "com/centreon/broker/database/table_max_size.hh"
```

Comment:
wrong path

Reply:
![image](https://user-images.githubusercontent.com/6324413/144279694-e5a1c851-4e3e-4555-ace8-d843eed61405.png)

Reply:
Yes it exists

Reply:
OK, the generator is not called before the bbdo compilation.